functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/customAssertions/CustomAssertions.java

```diff
@@ -0,0 +1,15 @@
+package org.arquillian.smart.testing.ftest.customAssertions;
+
+import org.arquillian.smart.testing.ftest.testbed.project.Project;
+import org.jboss.shrinkwrap.resolver.api.maven.embedded.BuiltProject;
+
+public class CustomAssertions {
```

Comment:
Do we really need this class?

Reply:
I am not sure. I thought it would be a good idea to have a common entry point for all our custom assertions.

Reply:
It is a good idea, but not only for this reason - see here http://joel-costigliola.github.io/assertj/assertj-core-custom-assertions.html#single-assertion-entry-point.

Please adjust accordingly.